schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INTEGER NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL NOT NULL,
+    species VARCHAR(100) NOT NULL,
 
```

Comment:
- [ ] [OPTIONAL] Good job so far! In my opinion it would be nice if you implement column called `species` to another table like that 👍🏼 

```sql
   ALTER TABLE animals ADD species varchar(255);
```

Reply:
thanks. i forgot to add that line in my code after altering the table directly.